src/sharing/index.js

```diff
@@ -12,25 +11,20 @@ const Sharing = NativeModules.Sharing ?? {
     null,
 };
 
-const goToSharing = (data: SharedData) => (dispatch, getState) => {
+const goToSharing = (data: SharedData) => {
   NavigationService.dispatch(navigateToSharing(data));
 };
 
-export const handleInitialShare = async (dispatch: Dispatch) => {
+export const handleInitialShare = async () => {
   const initialSharedData: SharedData | null = await Sharing.readInitialSharedContent();
   if (initialSharedData !== null) {
-    dispatch(goToSharing(initialSharedData));
+    goToSharing(initialSharedData);
   }
 };
 
 export class ShareReceivedListener {
-  dispatch: Dispatch;
   unsubs: Array<() => void> = [];
 
-  constructor(dispatch: Dispatch) {
-    this.dispatch = dispatch;
```

Comment:
```
sharing [nfc]: Stop taking dispatch, no longer used here

This module hasn't needed a Redux `dispatch` since 543c07c, when
we stopped using Redux for dispatching navigation actions.  So,
simplify by not taking one in the first place.
```

This is great!!